heron/statemgrs/src/python/statemanager.py

```diff
@@ -42,24 +42,12 @@ def name(self, newName):
     self.__name = newName
```

Comment:
you might want to change the example configuration files or given a commented example in

 heron/tools/config/src/yaml/tracker/heron_tracker.yaml